public/index.html

```diff
@@ -6,8 +6,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
```

Comment:
There is no need to include this file in `index.html`. We  already have `@fortawesome` and they are serving the same work. This means every page is loading this `font-awesome` regardless if that page needs it or not!